pkg/csvcopy/csvcopy.go

```diff
@@ -610,6 +623,42 @@ func (c *Copier) handleCopyError(ctx context.Context, db *sqlx.DB, batch Batch,
 
 	}
 
+	// Check if this is an encoding error
+	if isEncodingError(copyErr) && c.windows1252Handling {
```

Comment:
`c.windows1252Handling` should be the first one, so that we don't scan for encoding error if the handling is disabled.